docs/installation.md

```diff
@@ -0,0 +1,55 @@
+# Installation
+
+These instructions are likely in flux. If something looks out of date or is not working, please raise an issue.
+
+## Install dependencies
+
+Until a conda package is live, the core dependencies are listed in `devtools/conda-envs/minimal_env.yaml` and optional dependencies can be found in `devtools/conda-envs/test_env.yaml`.
```

Comment:
Are only the optional deps here, or is it the _minimal_env.yml_ and the other deps in test_env.yml